doc/articles/uno-publishing-windows.md

```diff
@@ -0,0 +1,100 @@
+---
+uid: uno.publishing.windows
+---
+
+# Publishing Your App for Windows
+
+## Preparing For Publish
+
+## Building your app
+
+Building your app requires using the msbuild command (`dotnet build` is not compatible as of WinAppSDK 1.5).
+
+### Side loading (without signing)
+
+#### Package the unsigned app
+
+Packaging the app without a code signature allows for installing the app on a machine without having to install the signer's certificate. This portion of the guide is derived from the [official Windows App SDK documentation](https://learn.microsoft.com/en-us/windows/msix/package/unsigned-package).
+
+To package your app:
+
+- Update the `Package.appxmanifest` file with the following `Identity` node to include the following `OID`:
+
+  ```xml
+  <Identity Name="MyApp"
+            Publisher="CN=AppModelSamples, OID.2.25.311729368913984317654407730594956997722=1"
+            Version="1.0.0.0" />
+  ```
+
+- Build your app in the folder of the app's `.csproj` by using the following command:
+
+    ```pwsh
+    msbuild /r /p:TargetFramework=net8.0-windows10.0.19041 /p:Configuration=Release /p:Platform=x64 /p:GenerateAppxPackageOnBuild=true /p:AppxBundle=Never /p:UapAppxPackageBuildMode=Sideloading /p:AppxPackageDir="C:/temp/output/" /p:AppxPackageSigningEnabled=false
+    ```
+
+> [!IMPORTANT]
+> Single package msix bundling is [not yet supported from msbuild the command line](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/single-project-msix?tabs=csharp#automate-building-and-packaging-your-single-project-msix-app).
+
+#### Install unsigned the app
+
+To install the app:
+
+- Start an elevated PowerShell command prompt (Search for PowerShell in the start menu, right-click on it then Run as administrator):
+- In the folder containing the `.msix` file, execute the following command :
+
+    ```pwsh
+    Add-AppPackage -AllowUnsigned ".\MyApp.appx"
+    ```
+
+For more information, see the [official documentation](https://learn.microsoft.com/en-us/windows/msix/package/unsigned-package#install-an-unsigned-package).
+
+### Side loading (with signing)
+
+#### Package signed the app
+
+This guide uses a self-signed certificate.
+
+To package your app:
+
+- Create a self-signed certificate:
+  - Open the solution in Visual Studio 2022
+  - Double-click on the `Package.appxmanifest` file
```

Comment:
@jeromelaban not sure if you can replicate this issue on your side
But with an Uno Platform application when double-clicking on the `Package.appxmanifest` file, most of the time I have this error. 
![image](https://github.com/unoplatform/uno/assets/16295702/afff091f-5097-4bc0-8458-68165dd173f9)
I tried with a WinUI application multiple times and I don't have this error.
______
Using Microsoft Visual Studio Professional 2022 (64-bit) - Current Version 17.9.6 for my tests

Reply:
Yes, I remember this happened to me as well in single project

Reply:
I'm not having that problem, tried on multiple machines, stable and preview. @MartinZikmund could you give it a try please?

Reply:
Hey @jeromelaban  , i did had this issue when i was trying to open the file
Templates version :- 5.2.31
VSIX version :- 5.2.48
VS version :- 17.9.6

Reply:
<img width="741" alt="Screenshot 2024-05-14 at 11 56 32 AM" src="https://github.com/unoplatform/uno/assets/90713481/f596e334-b18f-4a43-92a4-2b3cbf060662">

Reply:
This is the issue: https://developercommunity.visualstudio.com/t/Double-clicking-on-a-PackageAppxmanifes/10658683. Opening the file requires to be in ios, android, catalyst or windows target frameworks. The other target frameworks produce the error.

Reply:
VS Issue: https://developercommunity.visualstudio.com/t/Double-clicking-on-a-PackageAppxmanifes/10658683